modyn/tests/storage/internal/file_wrapper/test_csv_file_wrapper.py

```diff
@@ -0,0 +1,197 @@
+import os
```

Comment:
With Francesco's comment on the storage C++ rewrite in mind, can you please also add an integration test with csv? That would be amazing so we can test whether the C++ implementation will also work correctly :)

Reply:
OK, I can certainly do that. Regarding the implementation in C++: I have never written a line of code and it would be quite a big overhead to write this class in C++. Are there any updates on Olga? Maybe she can take care of it since it was her PR

Reply:
No need for you to do it in C++ now. Before merging C++, all integration tests will need to pass through and before that we won't merge. No need for you to write that, that will happen inside the C++ PR